tasks/species_typing/task_tbprofiler_output_parsing.wdl

```diff
@@ -18,45 +20,54 @@ task tbprofiler_output_parsing {
 
     ## Lookup Data Structures ##
 
-    # lookup dictionary - antimicrobial code to name
-    antimicrobial_dict = {"M_DST_B01_INH": "isoniazid", "M_DST_C01_ETO": "ethionamide",
+    # gene coverage as a dictionary
```

Comment:
coming back after going through, if these dicts below are meant to be global variables, make the variable names all capitalized, to make it aware later in the script it's a global variable